ros_controls.rolling-on-humble.repos

```diff
@@ -27,10 +27,10 @@ repositories:
     type: git
     url: https://github.com/ros-controls/ros2_control.git
     version: master
-  ros-controls/ros2_control_demos:
-    type: git
-    url: https://github.com/ros-controls/ros2_control_demos.git
-    version: master
+  # ros-controls/ros2_control_demos:
+  #   type: git
+  #   url: https://github.com/ros-controls/ros2_control_demos.git
+  #   version: master
```

Comment:
@christophfroehlich If I understood well, we are disabling the ros2_control_demos because they use gz_ros2_control right?